analyzers/tests/SonarAnalyzer.UnitTest/Rules/CryptographicKeyShouldNotBeTooShortTest.cs

```diff
@@ -25,35 +25,44 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class CryptographicKeyShouldNotBeTooShortTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<CryptographicKeyShouldNotBeTooShort>();
```

Comment:
`.AddReferences(GetAdditionalReferences())` is called for all tests. We could move it here. (It kind of contradics the comments with paths but I think paths are more test dependent than references)